packages/fxa-shared/test/db/models/auth/index.spec.ts

```diff
@@ -57,6 +58,24 @@ describe('auth', () => {
     });
   });
 
+  describe('getUidAndEmailByStripeCustomerId', () => {
```

Comment:
Since the new helper is a join on accounts and accountCustomers, I made a new top level describe block in this test. Is this the best place for it?